src/taxonomy/tag-list/data/selectors.js

```diff
@@ -0,0 +1,42 @@
+// @ts-check
```

Comment:
@rpenido From what I understand from [here](https://github.com/openedx/frontend-app-course-authoring/pull/645#discussion_r1377342836), these would not be considered selectors in the context of the other MFE modules. To avoid colliding with that context is why I have placed functions like this in `apiHooks.js`. But you can wait until https://github.com/openedx/frontend-app-course-authoring/pull/645 is approved to make those changes